api/filter/http/router.proto

```diff
@@ -18,4 +20,9 @@ message Router {
   // outbound calls and have child spans rooted at the same ingress
   // parent. Defaults to false.
   bool start_child_span = 2;
```

Comment:
Real quick, can you nuke this for now? I think it's not going to get used. If I actually need it I will add it back.